src/style.css

```diff
@@ -0,0 +1,104 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+* {
+  padding: 0;
```

Comment:
only box-sizing can be inside this block `*{}`

Reply:
ok, I'll remove it to body-block. is it allow?